### ISSUE TITLE
Options not showing

### ISSUE THREAD

**Issue by Alex-Mirov** [richardfrost/AdvancedProfanityFilter#326]
When using the extension for firefox on android the options not showing at all with 
version 2.23.1 even with stock settings.

P.S: On windows for firefox it works without any issues.

**Comment by richardfrost** [comment]
What version of Firefox mobile are you using?

**Comment by Alex-Mirov** [comment]
68.11.0

**Comment by TheModernWizard** [comment]
Can confirm. I am using Fennec F-Droid version 68.12.0 (Android), and every version of Advanced Profanity Filter from 2.21.0 to the present has a nonexistent settings page. 2.20.2.0 works great though, thanks for the addon.

**Comment by richardfrost** [comment]
Thanks for the details, and especially for tracking down the last working version @TheModernWizard. That really helped in troubleshooting. I think I have found the issue and should have a patch ready to go tonight.

**Comment by richardfrost** [comment]
I just pushed version [2.23.2](https://github.com/richardfrost/AdvancedProfanityFilter/releases/tag/2.23.2) which should fix the issue. I introduced a regex feature into the code that wasn't supported in older versions of Firefox for Android which broke those pages.

In this new version, I've added some additional output that should get shown to the page to help track down issues that might appear in the future, and then I re-visited that new feature and switched to a more performant alternative that should be supported on pretty much every browser now. Thanks again for reporting the issue, and for helping me track it down!

Once you get the update, let me know how it goes for you.
